docs/test_frameworks/xunit_net.md

```diff
@@ -43,7 +49,7 @@ be115f3df138   redis:7.0                   "docker-entrypoint.s…"   3 seconds
 
 ## Creating a shared test context
 
-Sometimes, creating and disposing of a test resource can be an expensive operation that you do not want to repeat for every test. By inheriting from the `ContainerFixture<TBuilderEntity, TContainerEntity>` class, you can share the test resource instance across all tests within the same test class.
+Sometimes, creating and disposing of a test resource can be an expensive operation that you do not want to repeat for every test. By inheriting from the `ContainerFixture<TBuilderEntity, TContainerEntity>` class, you can share the test resource instance across all tests within the same test class, or even within the assembly, if you use xUnit.net V3.
```

Comment:
```suggestion
Sometimes, creating and disposing of a test resource can be an expensive operation that you do not want to repeat for every test. By inheriting from the `ContainerFixture<TBuilderEntity, TContainerEntity>` class, you can share the test resource instance across all tests within the same test class, or even within the assembly.
```

Doesn't it work with v2 as well (using the [assembly annotation](https://github.com/testcontainers/testcontainers-dotnet/issues/1262#issuecomment-2817328066))?

Reply:
The second paragraph of https://xunit.net/docs/shared-context#assembly-fixture explicitly states it is new in v3, but there is https://github.com/xunit/samples.xunit/tree/main/v2/AssemblyFixtureExample which shows how to build a custom runner to support it in v2. The experience is IMHO not as good as it is in v3 though.

Are you still in favour of removing `if you use xUnit.net V3`? We could also go `if you use xUnit.net V3 or an appropriate extension`

Reply:
Ah, I think I mixed it up with collection fixtures. Thanks.